src/modules/pack.ts

```diff
@@ -347,7 +347,9 @@ export class PackModule extends ModuleWithRoles<PackContract> {
   public async setRestrictedTransfer(restricted = false): Promise<void> {
     await this.sendTransaction("setRestrictedTransfer", [restricted]);
   }
-
+  public async isRestrictedTransfer(): Promise<boolean> {
```

Comment:
We should rename all instances of `isRestrictedTransfer` to `isTransferRestricted`.